### PR TITLE
Etcd docs cleanup

### PR DESCRIPTION
**Pull request by olemarkus**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -81,10 +80,15 @@ nav:
     - Label management: "labels.md"
     - Secret management: "secrets.md"
     - Service Account Token Volume: "operations/service_account_token_volumes.md"
-    - Moving from a Single Master to Multiple HA Masters: "single-to-multi-master.md"
     - Working with Instance Groups: "tutorial/working-with-instancegroups.md"
     - Running kops in a CI environment: "continuous_integration.md"
     - etcd administration: "operations/etcd_administration.md"
```
Comment (rifelpet) [inline mkdocs.yml:85]: I think this line could be removed. Its already included in the submenu 2 lines below, as seen in [the preview](https://deploy-preview-8907--kubernetes-kops.netlify.com/operations/etcd_administration/)

```suggestion
```

Comment (olemarkus) [reply mkdocs.yml:85]: Thanks for catching that.

**Pull request merged**